ports/glutin/window.rs

```diff
@@ -1196,6 +1196,7 @@ impl WindowMethods for Window {
                 use glutin::MouseCursor;
 
                 let glutin_cursor = match c {
+                    CursorKind::Auto => MouseCursor::Default,
```

Comment:
Without this, we'll get a `pattern not covered` error.

I looked at the `glutin` crate and, if I understand correctly, it's used as the layer between Servo and the OS that lets changing the appearance of the mouse cursor, right? As there is no `MouseCursor::Auto` in that crate I used the `Default` variant for the time being. Shall we implement it there or is it okay the way it is? Also, can the OS even have a `Auto` variant of the mouse cursor? To me it seems It always has to be some specific type.

Reply:
About `pattern not covered` part, it's caused by removing the pattern of `CursorKind::Auto`.
When doing pattern matching, we need to cover all the patterns for the type so if you removed `CursorKind::Auto` in this pattern matching, compiler will complain about this pattern is not covered.
I don't understand the implementation part either so only reply to the pattern matching part!